packages/ui/src/Modal.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, ReactNode } from 'react'
```

Comment:
Should w also export `setModalAppElement`? http://reactcommunity.org/react-modal/accessibility/#app-element